lib/path.js

```diff
@@ -364,7 +369,8 @@ const win32 = {
 
     var tail;
     if (rootEnd < len)
```

Comment:
Can braces be added for the `if` and `else` here if we start having multi-line conditional bodies?